<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-5VHKQUJ" failures="4" tests="16" name="com.ae.qa.testcases.TenantsPageTest" time="601.620" errors="1" timestamp="30 Jan 2023 08:09:43 GMT" skipped="0">
  <testcase classname="com.ae.qa.testcases.TenantsPageTest" name="endReport" time="17.105">
    <error message="javax.mail.AuthenticationFailedException: 535 5.7.139 Authentication unsuccessful, the user credentials were incorrect. [PN2PR01CA0147.INDPRD01.PROD.OUTLOOK.COM 2023-01-30T08:09:43.225Z 08DB023907B03DA6]
" type="java.lang.RuntimeException">
      <![CDATA[java.lang.RuntimeException: javax.mail.AuthenticationFailedException: 535 5.7.139 Authentication unsuccessful, the user credentials were incorrect. [PN2PR01CA0147.INDPRD01.PROD.OUTLOOK.COM 2023-01-30T08:09:43.225Z 08DB023907B03DA6]

at com.ae.qa.util.SendMail.sendExtentReport(SendMail.java:79)
at com.ae.qa.base.TestBase.endReport(TestBase.java:218)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
at org.testng.TestRunner.afterRun(TestRunner.java:725)
at org.testng.TestRunner.run(TestRunner.java:509)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
Caused by: javax.mail.AuthenticationFailedException: 535 5.7.139 Authentication unsuccessful, the user credentials were incorrect. [PN2PR01CA0147.INDPRD01.PROD.OUTLOOK.COM 2023-01-30T08:09:43.225Z 08DB023907B03DA6]

at com.sun.mail.smtp.SMTPTransport$Authenticator.authenticate(SMTPTransport.java:965)
at com.sun.mail.smtp.SMTPTransport.authenticate(SMTPTransport.java:876)
at com.sun.mail.smtp.SMTPTransport.protocolConnect(SMTPTransport.java:780)
at javax.mail.Service.connect(Service.java:388)
at javax.mail.Service.connect(Service.java:246)
at javax.mail.Service.connect(Service.java:195)
at javax.mail.Transport.send0(Transport.java:254)
at javax.mail.Transport.send(Transport.java:124)
at com.ae.qa.util.SendMail.sendExtentReport(SendMail.java:76)
... 28 more
]]>
    </error>
  </testcase> <!-- endReport -->
  <testcase classname="com.ae.qa.testcases.TenantsPageTest" name="validateAdvSearchForOrgCodeEqualToTenantPageTest" time="39.389"/>
  <testcase classname="com.ae.qa.testcases.TenantsPageTest" name="validateAdvSearchForOrgCodeNotEqualToTenantPageTest" time="45.410"/>
  <testcase classname="com.ae.qa.testcases.TenantsPageTest" name="validateAdvSearchForOrgCodeIsLikeTenantPageTest" time="31.099">
    <failure message="Tenant not created successfully expected [Tenant created successfully] but found [Duplicate org code [ICICI98].]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Tenant not created successfully expected [Tenant created successfully] but found [Duplicate org code [ICICI98].]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at com.ae.qa.pages.TenantsPage.addNewTenants(TenantsPage.java:163)
at com.ae.qa.pages.TenantsPage.validateAdvSearchForOrgCodeIsLike(TenantsPage.java:681)
at com.ae.qa.testcases.TenantsPageTest.validateAdvSearchForOrgCodeIsLikeTenantPageTest(TenantsPageTest.java:201)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:831)
]]>
    </failure>
  </testcase> <!-- validateAdvSearchForOrgCodeIsLikeTenantPageTest -->
  <testcase classname="com.ae.qa.testcases.TenantsPageTest" name="validateAdvSearchOrgBeginsWithTenantPageTest" time="34.784"/>
  <testcase classname="com.ae.qa.testcases.TenantsPageTest" name="validateAdvSearchOrgEndsWithTenantPageTest" time="38.129"/>
  <testcase classname="com.ae.qa.testcases.TenantsPageTest" name="validateAdvSearchNameEqualToTenantPageTest" time="29.005"/>
  <testcase classname="com.ae.qa.testcases.TenantsPageTest" name="validateAdvSearchNameNotEqualTenantPageTest" time="31.992">
    <failure message="Tenant not created successfully expected [Tenant created successfully] but found [Duplicate org code [ABC081].]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Tenant not created successfully expected [Tenant created successfully] but found [Duplicate org code [ABC081].]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at com.ae.qa.pages.TenantsPage.addNewTenants(TenantsPage.java:163)
at com.ae.qa.pages.TenantsPage.validateAdvSearchNameNotEqual(TenantsPage.java:749)
at com.ae.qa.testcases.TenantsPageTest.validateAdvSearchNameNotEqualTenantPageTest(TenantsPageTest.java:242)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:831)
]]>
    </failure>
  </testcase> <!-- validateAdvSearchNameNotEqualTenantPageTest -->
  <testcase classname="com.ae.qa.testcases.TenantsPageTest" name="validateAdvSearchNameIsLikeTenantPageTest" time="29.336">
    <failure message="Tenant not created successfully expected [Tenant created successfully] but found [Duplicate org code [ABC8KG1].]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Tenant not created successfully expected [Tenant created successfully] but found [Duplicate org code [ABC8KG1].]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at com.ae.qa.pages.TenantsPage.addNewTenants(TenantsPage.java:163)
at com.ae.qa.pages.TenantsPage.validateAdvSearchNameIsLike(TenantsPage.java:769)
at com.ae.qa.testcases.TenantsPageTest.validateAdvSearchNameIsLikeTenantPageTest(TenantsPageTest.java:252)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:831)
]]>
    </failure>
  </testcase> <!-- validateAdvSearchNameIsLikeTenantPageTest -->
  <testcase classname="com.ae.qa.testcases.TenantsPageTest" name="validateAdvSearchNameBeginWithTenantPageTest" time="28.067">
    <failure message="Tenant not created successfully expected [Tenant created successfully] but found [Duplicate org code [HDFC110].]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Tenant not created successfully expected [Tenant created successfully] but found [Duplicate org code [HDFC110].]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at com.ae.qa.pages.TenantsPage.addNewTenants(TenantsPage.java:163)
at com.ae.qa.pages.TenantsPage.validateAdvSearchNameBeginWith(TenantsPage.java:789)
at com.ae.qa.testcases.TenantsPageTest.validateAdvSearchNameBeginWithTenantPageTest(TenantsPageTest.java:262)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:831)
]]>
    </failure>
  </testcase> <!-- validateAdvSearchNameBeginWithTenantPageTest -->
  <testcase classname="com.ae.qa.testcases.TenantsPageTest" name="validateAdvSearchNameEndWithTenantPageTest" time="35.607"/>
  <testcase classname="com.ae.qa.testcases.TenantsPageTest" name="validateCreatedEqualToTenantPageTest" time="42.271"/>
  <testcase classname="com.ae.qa.testcases.TenantsPageTest" name="validateCreatedBeforeTenantPageTest" time="41.311"/>
  <testcase classname="com.ae.qa.testcases.TenantsPageTest" name="validateCreatedAfterTenantPageTest" time="56.485"/>
  <testcase classname="com.ae.qa.testcases.TenantsPageTest" name="validateCreatedInBetweenTenantPageTest" time="48.063"/>
  <testcase classname="com.ae.qa.testcases.TenantsPageTest" name="validateCreatedNotInBetweenTenantPageTest" time="53.567"/>
</testsuite> <!-- com.ae.qa.testcases.TenantsPageTest -->
